README.md

```diff
@@ -1,48 +1,51 @@
 # Frontend assignment
 
-[Royal HaskoningDHV][]
+_For [Royal HaskoningDHV][]._
 
 * [Overview][]
 * [Acceptance criteria][]
-* [Optionals][]
+* [Technical criteria][]
 
 ## Overview
 [Overview]: #overview
 
-Use the framework of your choice to build an application that gets the current
-location, and then uses said location to fetch and display the current weather
-from the [OpenWeatherMap API][].
+Build an application that gets the current location, and then uses said location
+to fetch and display the current weather from the [OpenWeatherMap API][].
 
 ## Acceptance criteria
 [Acceptance criteria]: #acceptance-criteria
 
-The following things are stricly necessary for the application to be considered
-complete:
-
-* Errors need to be handled properly.
 * At least the following needs to be displayed as a result:
   * The location name.
   * The current temperature.
   * A short description of the current weather (_Clear_, _Foggy_, etc).
 * It should be possible to re-fetch/refresh the result.
-* The application should be available through [Git][].
-* Documentation on how to set up and run the application locally.
+* The application should be responsive.
+* _Optional_: Three day weather forecast for current location.
+* _Optional_: Daily notifications of weather in current location.
 
-## Optionals
-[Optionals]: #optionals
+## Technical criteria
+[Technical criteria]: #technical-criteria
 
-Although not required, the following items will be advantageous if added:
-
-* Testing.
-* TypeScript.
-* Dockerization.
-* Linting ([Eslint][] or similar).
-* Formatting ([Prettier][] or similar).
+* Framework of your choice (we use [React][]).
+* Errors need to be handled properly.
+* The application should be available through [Git][].
```

Comment:
Candidates often don't want to expose a public repo to avoid making known that they are applying for a Job, and GitHub does not have private link-sharing. It may be useful to either add a GitHub username they should share the assignment with and/or inform them how they can [bundle and e-mail](https://statagroup.com/articles/git-bundle) their git repo.

Reply:
Perfect, makes complete sense, added two accounts that repos can be shared with.